Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:7-alpine
+LABEL description="Alpine image to build nativfier apps"
+
+### Install wine depedency
+RUN apk add --no-cache \
+    wine \
+    freetype \
+    ### make symbolic link to use `wine`
+    && ln -s /usr/bin/wine64 /usr/bin/wine
+    
+# Add sources
+ADD . /nativefier
```

Comment:
I think it's better practice to use `COPY` instead of `ADD` https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy